kayenta-integration-tests/src/test/java/com/playtika/services/infra/kayenta/configuration/EmbeddedPrometheusBootstrapConfiguration.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Playtika
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.playtika.services.infra.kayenta.configuration;
+
+import static com.playtika.test.common.utils.ContainerUtils.containerLogsConsumer;
+
+import java.time.Duration;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.testcontainers.Testcontainers;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.utility.MountableFile;
+
+@Slf4j
+@Configuration
+public class EmbeddedPrometheusBootstrapConfiguration {
```

Comment:
please add additional submodule to testcontainers-spring-boot

Reply:
okay